lib/sarif/gosec_sarif.rb

```diff
@@ -5,6 +5,7 @@ class GosecSarif < BaseSarif
     def initialize(scan_report)
       super(scan_report)
       @uri = GOSEC_URI
+      @issues = Set.new
```

Comment:
Minor nit... I think you'd have a small readability win if you move the parsing of the scan report into its own method.

That would thin out the initialize method to something like

```
def initialize(scan_report)
  super(scan_report)
  @uri = GOSEC_URI
  parse_scan_report!(scan_report)
end
```